src/pepr/uds-operator-config/values.yaml

```diff
@@ -5,3 +5,5 @@ operator:
   UDS_SINGLE_TEST: "###ZARF_VAR_UDS_SINGLE_TEST###"
   UDS_LOG_LEVEL: "###ZARF_VAR_UDS_LOG_LEVEL###"
   AUTHSERVICE_REDIS_URI: "###ZARF_VAR_AUTHSERVICE_REDIS_URI###"
+  # Tweak Pepr watch config to react to dropped connections faster
+  PEPR_LAST_SEEN_LIMIT_SECONDS: "60"
```

Comment:
Open question - I think it makes sense to update the default here but we could make this zarf var and update the default or just make it zarf var and update the deploy time config in other environments where we have experienced this issue.

Reply:
```suggestion
  # Tweak Pepr watch config to react to dropped connections faster
  PEPR_LAST_SEEN_LIMIT_SECONDS: "300"
```

Reply:
Updating to default to the default pepr values, this can be helm overridden at the bundle layer for testing in staging.